MdePkg/Library/BaseMemoryLibOptPei/BaseMemoryLibOptPei.inf

```diff
@@ -27,19 +27,21 @@
 
 [Sources]
   MemLibInternals.h
+  ScanMem64Wrapper.c
+  ScanMem32Wrapper.c
+  ScanMem16Wrapper.c
+  ScanMem8Wrapper.c
+  ZeroMemWrapper.c
+  CompareMemWrapper.c
+  SetMem64Wrapper.c
+  SetMem32Wrapper.c
+  SetMem16Wrapper.c
+  SetMemWrapper.c
+  CopyMemWrapper.c
+  IsZeroBufferWrapper.c
+  MemLibGuid.c

```

Comment:
This part of the change looks like it is adding source files to the INF.  Not removing duplicates.  Please explain the reason for this change.

Reply:
the same .C source were including in both IA32 and X64.
removed those .C source from IA32 and X64 and moved to common source section header.